mingw-w64-arrow/PKGBUILD

```
@@ -3,14 +3,15 @@
 _realname=arrow
 pkgbase=mingw-w64-${_realname}
 pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
-pkgver=8.0.0
+pkgver=9.0.0
 pkgrel=1
 pkgdesc="Apache Arrow is a cross-language development platform for in-memory data (mingw-w64)"
 arch=("any")
 url="https://arrow.apache.org/"
 license=("Apache-2.0")
 depends=("${MINGW_PACKAGE_PREFIX}-aws-sdk-cpp"
          "${MINGW_PACKAGE_PREFIX}-brotli"
+         "${MINGW_PACKAGE_PREFIX}-bzip2"
```

https://github.com/apache/arrow/blob/master/ci/scripts/PKGBUILD#L29, see also https://github.com/apache/arrow/commit/3ac0959ac168caebb19dfbfbc8881323e694a4ae#diff-201c689fdfbc01345f8bbdee6cb0c4c73cb3968702f2c151d48baac32e1527e9

```suggestion
         "${MINGW_PACKAGE_PREFIX}-bzip2"
         "${MINGW_PACKAGE_PREFIX}-curl" # for google-cloud-cpp bundled build
```